controllers/flinkcluster_submit_job_script.go

```diff
@@ -66,7 +66,7 @@ function wait_for_job() {
 	while true; do
 		echo -e "\nWaiting for job to finish..."
 		list_jobs
-		if list_jobs | grep -e "(FINISHED)" -e "(FAILED)"; then
+		if list_jobs | grep -e "(FINISHED)" -e "(FAILED)" -e "(CANCELLED)"; then
```

Comment:
I tested, seems it should be `CANCELED` instead of `CANCELLED`.

Reply:
yes, It's my fault